script.js

```diff
@@ -0,0 +1,64 @@
+
+var game = {
+  els: {
+    playArea: $(".play-area"),
+    resetButton: $(".reset"),
+    nextPlayer: $(".next-player")
+  },
+  playersTurn: 1,
+  players: [],
+  tileGrid: [],
+
+  buildGameboard: function() {
+    for (var i = 0; i < 9; i++) {
+      var newTile = $("<div></div>").addClass("tile");
+      this.els.playArea.append(newTile);
+    }
+
+    this.els.tiles = $(".tile");
+    this.els.nextPlayer.attr("class", "next-player").addClass("x").html("<span>X</span>");
+  },
+
+  makeClickable: function() {
+    var self = this;
+    for (var i = 0; i < self.els.tiles.length; i++) {
+      self.els.tiles.eq(i).on("click", self.displayXO);
+    }
+  },
+
+  groupTiles: function() {
+    for (var i = 0, j = 0; i < 3; i++) {
+      var group = this.els.tiles.splice(j, 3);
+      this.tileGrid.push(group);
+    }
+    console.log(this.tileGrid);
+  },
+
+  displayXO: function(event) {
+    if (game.playersTurn % 2 !== 0) {
+      $(event.target).addClass("x").html("<span>X</span>").off("click");
+      game.els.nextPlayer.attr("class", "next-player").addClass("o").html("<span>O</span>");
+      game.playersTurn++;
+    } else {
+      $(event.target).addClass("o").html("<span>O</span>").off("click");
+      game.els.nextPlayer.attr("class", "next-player").addClass("x").html("<span>X</span>");
+      game.playersTurn++;
+    }
+  },
+
```

Comment:
How could we implement a `checkWinner` functionality? Feeling like we want to store info about each player's moves